lib/index.d.ts

```diff
@@ -101,6 +101,7 @@ declare module 'node-mocks-http' {
         body?: Body;
         query?: Query;
         files?: Files;
+        ip?: string;
 
```

Comment:
Should `ips?: [string]` be added here as well?

Reply:
It depends on the strategy that we want to implement:
- Simple configuration: Just set `ip` value and we calculate the `ips` property automatically. `ips` always include the same ip than `ip` property.
- Advanced configuration: You can configure both properties independently with different values. `ips` is empty if not configured explicitly (despite `ip` will be always defined).

The second approach is, strictly, the right one as [express says in req.ips documentation](https://expressjs.com/en/api.html#req.ips):
> **req.ips**
> When the trust proxy setting does not evaluate to false, this property contains an array of IP addresses specified in the X-Forwarded-For request header. Otherwise, it contains an empty array.

I chose the first one because it was simpler for library users, but I can switch to the second one if we both agree.

Reply:
Let's keep it simple.